CONTRIBUTING.md

```diff
@@ -30,13 +30,21 @@ The public key can be found on [coblox.tech](https://coblox.tech/security_coblox
 
 Thank you for taking the time to make COMIT safer!
 
-
 ## Feature Requests
 
 To request a change on the way COMIT works, please head over to the [RFCs repository](https://github.com/comit-network/rfc) and checkout the [RFC contributing guidelines](https://github.com/comit-network/rfc/blob/master/CONTRIBUTING.md).
  
 For changes related to our implementation, please [create a feature request](/issues/new/choose) and follow the GitHub template.
 
+## Fixing an issue
+
+If you wish to pick up an existing issue and resolve it, please be aware of the following labels:
+- ✅ [`groomed`](https://github.com/comit-network/comit-rs/labels/groomed): it has been discussed and should contain enough details to get started
```

Comment:
❓ Are these emojis helpful? How are the labels associated with them?

Reply:
It means "yes, you can pick up issues labelled with X" "no, you should not pick up issues labelled with Y".

Happy to remove if it's confusing.

Reply:
I like the emojis.